frontend/src/components/workspaces/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -0,0 +1,155 @@
+import { FC } from 'react';
+import { Col, Table } from 'antd';
+import { TemplatesTableRow } from '../TemplatesTableRow';
+import { DownOutlined, RightOutlined } from '@ant-design/icons';
+import { useState } from 'react';
+
+export interface ITemplatesTableProps {
+  templates: Array<{
+    id: string;
+    name: string;
+    gui: boolean;
+    instances: Array<{
+      id: number;
+      name: string;
+      ip: string;
+      status: boolean;
+    }>;
+  }>;
+  role: string;
+  createInstance: (id: string) => void;
+  destroyInstance: (idInstance: number, idTemplate: string) => void;
+  editTemplate: (id: string) => void;
+  deleteTemplate: (id: string) => void;
+}
+
+const TemplatesTable: FC<ITemplatesTableProps> = ({ ...props }) => {
+  const {
+    templates,
+    role,
+    createInstance,
+    editTemplate,
+    deleteTemplate,
+  } = props;
+
+  /**
+   * Our Table has just one column which render all rows using a component TemplateTableRow
+   */
+  /* eslint-disable */
```

Comment:
Disable only the rules you need disabled `react/no-multi-comp`

Reply:
Changed applied [here](https://github.com/netgroup-polito/CrownLabs/pull/460/commits/77c8d7a732e466080c9e305ba922259e2ecdaa42)